pkg/reconciler/openshift/tektonresult/extension.go

```diff
@@ -141,3 +150,79 @@ func filterAndTransform() client.FilterAndTransform {
 		return manifest, nil
 	}
 }
+
+// injectBoundSAToken adds a sa token projected volume to the Results Deployment
+func injectBoundSAToken(comp v1alpha1.TektonComponent) mf.Transformer {
```

Comment:
this will be added for all the cases. Can there will be cases where users does not want to do this?

Reply:
Yes, it will be added for all users. It's a conscious decision on my part.
It doesn't matter whether the user requires this or not. This is something which is used when we have WIF. Even if it's not WIF, then we are just getting a SA token mounted.
We just don't allow removing this token which is provided by OpenShift in all clusters. Or we can have just a configuration to enable or disable this but every configuration adds overhead towards default installation.

Reply:
https://docs.openshift.com/container-platform/4.11/authentication/managing_cloud_provider_credentials/cco-mode-sts.html

Reply:
In this case results API server requires s3 or GCS bucket for logging.

Reply:
thanks @piyush-garg @khrm 
I am fine with this changes

Reply:
as discussed on chat, this will not impact and results should work fine for both type of usersm using WIF or not